_1327/minutes/views.py

```diff
@@ -1,41 +1,24 @@
-from collections import OrderedDict
-
-from django.conf import settings
 from django.contrib.auth.models import Group
 from django.core.exceptions import ObjectDoesNotExist, SuspiciousOperation
 from django.shortcuts import render
 
+from guardian.shortcuts import get_objects_for_group
+
 from _1327.minutes.models import MinutesDocument
 
 
-def list(request, groupid=None):
-	if groupid:
-		groupid = int(groupid)
-		try:
-			Group.objects.get(id=groupid)
-		except ObjectDoesNotExist:
-			raise SuspiciousOperation
-	groups = {}
-	minutes = MinutesDocument.objects.order_by('-date')
+def list(request, groupid):
```

Comment:
please explain what this function does and why we filter for the change permission